PySDM/backends/impl_numba/methods/moments_methods.py

```diff
@@ -31,59 +45,122 @@ def moments_body(
                             multiplicity[i]
                             * weighting_attribute[i] ** weighting_rank
                             * attr_data[i] ** ranks[k]
-                        )
+                        ),
+                    )
+        if not skip_division_by_m0:
+            for c_id in range(moment_0.shape[0]):
+                for k in range(ranks.shape[0]):
+                    moments[k, c_id] = (
+                        moments[k, c_id] / moment_0[c_id] if moment_0[c_id] != 0 else 0
                     )
-        for c_id in range(moment_0.shape[0]):
```

Comment:
Example showing tht diff is not against current master: 
https://github.com/atmos-cloud-sim-uj/PySDM/blob/f69124d75f2419dbc82ab14742cac5fa06433fc3/PySDM/backends/impl_numba/methods/moments_methods.py#L37

How to fix it?

Reply:
Ah, so you mean that is should show that `for c_id in range(moment_0.shape[0]):` is deleted from line _38_ not line 36?

Reply:
yes, and the recently added line 37 from master is apparently not taken into account what makes it more puzzling to go through the long file list :( - is there any way to force github to compare with master? why does it not do it? Thanks